src/decomon/layers/decomon_layers.py

```diff
@@ -136,7 +136,7 @@ def build(self, input_shape: List[tf.TensorShape]) -> None:
         self.built = True
 
     def get_backward_weights(self, inputs: List[tf.Tensor], flatten: bool = True) -> Tuple[tf.Tensor, tf.Tensor]:
-
+        # DEPRECATED
```

Comment:
Rather than putting a "deprecated" comment (not visible by the user when using it by the way), i would rather remove the method as it is not used by the code now and was not meant to be used outside of `BackwardConv2D.call()`